src/data_processing/wiki_to_netflix.py

```diff
@@ -0,0 +1,136 @@
+import requests
+import csv
+
+base_dir = os.path.join(os.path.dirname(__file__), '../data/')
+user_agent = 'Noisebridge MovieBot 0.0.1/Audiodude <audiodude@gmail.com>'
+
+missing_count = 0
+num_rows = 42
+
+# Reading netflix text file
+def read_netflix_txt(txt_file):
+    with open(txt_file, "r", encoding = "ISO-8859-1") as netflix_data:
+        netflix_list = [line.rstrip().split(',', 2) for line in netflix_data.readlines()[0:num_rows]]
+    return(netflix_list)
+
+# Writing netflix csv file
+def create_netflix_csv(csv_name, data_list):   
+    with open(csv_name, 'w') as netflix_csv:
+        csv.writer(netflix_csv).writerows(data_list)
+
+# Extracting movie info from Wiki data
+def wiki_feature_info(data, key):
+    if len(data['results']['bindings']) < 1 or key not in data['results']['bindings'][0]:
+        return 'NA'
+    if key == 'genreLabel':
+        return list({d['genreLabel']['value'] for d in data['results']['bindings'] if 'genreLabel' in d})
+    return data['results']['bindings'][0][key]['value'].split('/')[-1]
+
+# Adding movie info to netflix csv file
+def add_movie_info_to_csv(data_csv, movies, genres, directors):
```

Comment:
This doesn't look like it's used anywhere. And I would be worried if it was, because it is very inefficient to read a CSV file and change one row and write it back out.

Reply:
Yeah, this is also a holdover from before the refactor which isn't currently being used. It has been deleted